contracts/Polymorph.sol

```diff
@@ -57,17 +60,18 @@ contract Polymorph is IPolymorph, ERC721PresetMinterPauserAutoId, BMath, Reentra
     }
 
     function mint() public override payable nonReentrant {
+        require(msg.value >= polymorphPrice, "Not enough funds");
```

Comment:
I dont think you need to check this, as both the `.call` and `.sub` will fail if less than the polymorph price

Reply:
Done!